### ISSUE TITLE
Start Page for Lapce

### ISSUE THREAD

**Issue by prophet1996** [lapce/lapce#1167]
## Is your feature request related to a problem?
Want to see  a start page like vscode/intellij, which has all the recent projects listed 

## Additional context
![excali](https://user-images.githubusercontent.com/16836510/189594706-2010ca46-c45d-4ce8-bb84-1debf74fc47e.png)

**Comment by ghishadow** [comment]
until this is implemented, you can use Recent Workspaces 
![image](https://user-images.githubusercontent.com/9583775/189598137-338f2e14-a605-45f6-8247-5ea7b573753c.png)

**Comment by prophet1996** [comment]
> > ## Is your feature request related to a problem?
> > Want to see a start page like vscode/intellij, which has all the recent projects listed
> > ## Additional context
> > ![excali](https://user-images.githubusercontent.com/16836510/189594706-2010ca46-c45d-4ce8-bb84-1debf74fc47e.png)
> 
> > Extensions is not available right now check this repo #558
> 
> i think Recents are very Useful when we start app we can choose our recent projects. 

yes it's a kind of a feature you don't think about much but pretty useful :)
